azure-pipelines.yml

```diff
@@ -339,6 +347,22 @@ stages:
           name: Build
           displayName: Build
           condition: succeeded()
+        - script: |
+            rm -rf $HOME/.yarn
+            curl -o- -L https://yarnpkg.com/install.sh | bash
+            set -e
+            export PATH="$HOME/.yarn/bin:$HOME/.config/yarn/global/node_modules/.bin:$PATH"
+            sudo apt-get update
+            sudo apt-get install ia32-libs
```

Comment:
[Turns out this isn't available on Ubuntu](https://askubuntu.com/a/578176), which [is why it's failing](https://dev.azure.com/dnceng/public/_build/results?buildId=502783&view=logs&j=5790f492-ad1a-5ccb-ede2-0470aee98b0b&t=0b04d4f9-1632-59f0-7bb8-6af5e423b7fe&l=107)

Reply:
Yup. I'm trying a bunch of things. Definitely not ready for review yet 😄

Reply:
If we can even just get tests running on a single platform that'd be sufficient for now.

Reply:
Exactly what I'm trying. I've given up on macOS for now because I tried a bunch of things but I can't make VSCode use the dotnet that I've activated. So, Linux is my only hope but it is complaining about `libgdk_pixbuf-2.0.so.0` not being found. The above line was my failed attempt at fixing that issue.

Reply:
```
Downloading VS Code insiders from https://update.code.visualstudio.com/latest/linux-x64/insider
Downloaded VS Code insiders into .vscode-test/vscode-insiders
/__w/1/s/src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.FunctionalTest/.vscode-test/vscode-insiders/VSCode-linux-x64/bin/../code-insiders: error while loading shared libraries: libgdk_pixbuf-2.0.so.0: cannot open shared object file: No such file or directory
/__w/1/s/src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.FunctionalTest/.vscode-test/vscode-insiders/VSCode-linux-x64/code-insiders: error while loading shared libraries: libgdk_pixbuf-2.0.so.0: cannot open shared object file: No such file or directory

Exit code:   127
```
This is the exact failure.

Reply:
@octref might be able to shed some insight

Reply:
I have never seen that. Works fine for me https://dev.azure.com/vscode/vscode-test/_build/results?buildId=36000&view=results with this definition: https://github.com/microsoft/vscode-test/blob/master/sample/azure-pipelines.yml